packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2377,7 +2377,7 @@ const schema: SchemaType<DbUser> = {
   },
   subforumPreferredLayout: {
     type: String,
-    allowedValues: subforumLayouts,
+    allowedValues: Array.from(subforumLayouts),
```

Comment:
This was causing an error every time `yarn generate` was run. I had generated the types before changing `subforumLayouts` to be a TupleSet